.github/workflows/openapi-update-generate-new-sdk.yml

```diff
@@ -1,9 +1,9 @@
 name: Update bundled openapi definition & Generate SDK
 
 on:
-  push:
-    branches:
-      - main
+ # push:
+ #   branches:
+ #     - main 
```

Comment:
Is this for testing or should this remain commented out?

Reply:
Good notice, I will uncommented it after merge because it will create changes that not needed for testing.